src/custom-hooks/useLocalStorage.ts

```diff
@@ -0,0 +1,13 @@
+import { useState, useEffect } from "react";
+
+export const useLocalStorage = (key, defaultValue) => {
```

Comment:
Should the parameters have types?

Reply:
I think so. I'll check how hooks work with types.

Reply:
added types

Reply:
Thanks!